libs/api/books/src/lib/reading-list.service.ts

```diff
@@ -13,32 +13,28 @@ export class ReadingListService {
   }
 
   async addBook(b: Book): Promise<void> {
-    this.storage.update(list => {
+    this.storage.update((list) => {
```

Comment:
Avoid such changes in final MR and format your newly added code before you commit